### ISSUE TITLE
gazebo-7 and qwt-6: non-existing USE flag

### ISSUE THREAD

**Issue by flabrosse** [ros/ros-overlay#652]
qwt-6, required for gazebo-7, required for ros-lunar, has dropped the qt5 USE flag.  emerge complains about this:

```
emerge -av gazebo

These are the packages that would be merged, in order:

Calculating dependencies... done!

emerge: there are no ebuilds built with USE flags to satisfy "x11-libs/qwt:6=[qt5]".
!!! One of the following packages is required to complete your request:
- x11-libs/qwt-6.1.3-r2::gentoo (Missing IUSE: qt5)
(dependency required by "sci-electronics/gazebo-7.11.0::ros-overlay" [ebuild])
(dependency required by "gazebo" [argument])`

Because of this (and other issues), gazebo cannot be emerged.
```

**Comment by allenh1** [comment]
Hmm.... That's certainly a problem!

Would you mind doing a PR to replace/remove that `USE` flag from the ebuild?

**Comment by flabrosse** [comment]
Assuming PR means private repo, I have just done that and it now does not complain about this.

This however highlights an interesting problem.  Gazebo-7 wants qt4, which has recently been masked.  That and all the other problems I've been having with compiling gazebo-7 (see other issues), makes me say that I'm going to give up on gazebo on gentoo...

Time ROS upgrades its integration to recent versions of gazebo!

**Comment by allenh1** [comment]
> Assuming PR means private repo

I actually meant pull request.

> I'm going to give up on gazebo on gentoo

Maybe we should look into Gazebo 8? I also need to make a Gazebo 9 ebuild. I'm rather behind with Gazebo...

Don't give you just yet!

**Comment by flabrosse** [comment]
Pull request?  I still have to learn how to use git!...

Problem with gazebo 8 or 9 is that ROS still wants to use 7.

**Comment by allenh1** [comment]
> Pull request? I still have to learn how to use git!...

No worries. On your fork you should see "Pull Request" somewhere. After you've got some fixes, just click that button to get them merged in here.

> Problem with gazebo 8 or 9 is that ROS still wants to use 7.

That's very version-dependent. Kinetic is running 7, I believe Lunar is running 7, and melodic runs 9 (see [here](http://gazebosim.org/tutorials?tut=ros_wrapper_versions#UsingthedefaultGazeboversionforaROSdistribution) for more).

You can, however, try to use 8 with Lunar (if I remember -- I may be wrong).

**Comment by flabrosse** [comment]
I don't fork, I just use layman to get the overlay (maybe it does that behind the scene, I don't know).

So far I've been using Lunar, but for a current project with an external organisation we may well have to revert to kinetic.  I don't think melodic is even being considered by them.  We'll see.

**Comment by allenh1** [comment]
Fixed by #684
